<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVN User Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	color:  rgb(187, 187, 187);
	background-color: rgb(68, 68, 68);
}

QLineEdit
{
	border: none;
	border-width: 2px;
	color:  rgb(187, 187, 187);
	background-color: rgb(43, 43, 43);
	padding: 2px;	
}

QPlainTextEdit
{
	border: none;
	color:  rgb(187, 187, 187);
	background-color: rgb(43, 43, 43);	
}

QPushButton
{
	border-style: outset;
	color:  rgb(187, 187, 187);
	background-color: rgb(93, 93, 93);
	padding: 5px;
}

QPushButton:hover
{
   background-color: rgb(112, 112, 112);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_username">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="str_username">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_password">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="str_password">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_ok">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
